starfish_py/config.py

```diff
@@ -44,23 +42,40 @@ class Config(configparser.ConfigParser):
     Create a new config instance, using a config file, dictionary of values or argument list (kwargs)
 
     :param filename: Filename of the config file to load.
+    :type filename: str or None
     :param contracts_path: path to the contract files ( artifacts ).
+    :type contracts_path: str or None
     :param keeper_url: url to the keeper node ( http://localhost:8545 ).
+    :type keeper_url: str or None
```

Comment:
(This is a general observation on this entire block of comments.)

Many of the existing services are referenced by their version names (e.g., keeper, Aquarius, etc.) as opposed to defining  the generic service provided and then, maybe, a parenthetical reference to an existing implementation.
For example,

**:param aquarius_url: url to metadata service (e.g., using the Aquarius metadata service this would be _http://localhost:5000_)**

Since it's quite possible that major version releases will imply name changes, making this generic now will save us the hassle of having to come back through and do it later.